.github/workflows/fips-compliance.yml

```
@@ -0,0 +1,55 @@
+    - name: Run FIPS compliance check
+      run: |
+        /tmp/check-payload/check-payload scan local --path /tmp/container-fs
+    
+    - name: Upload build artifacts on failure
+      if: failure()
+      uses: actions/upload-artifact@v4
+      with:
+        name: fips-compliance-logs
+        path: |
+          kube-auth-proxy-fips.tar
+        retention-days: 7
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)